app/data/schemas/frn.js

```diff
@@ -0,0 +1,24 @@
+const Joi = require('joi')
+
+module.exports = {
+  frn: Joi.number().integer().min(1000000000).max(9999999999).required()
+    .error(errors => {
+      errors.forEach(err => {
+        switch (err.code) {
+          case 'number.min':
+            err.message = 'The FRN is too short, it must be 10 digits'
+            break
+          case 'number.max':
+            err.message = 'The FRN is too long, it must be 10 digits'
+            break
+          case 'number.base':
+            err.message = 'The FRN must be a 10 digit number'
+            break
+          default:
+            err.message = 'The FRN is invalid, it must be 10 digits'
+            break
+        }
+      })
```

Comment:
There's a better way of doing as according to Joi we're doing it wrong.

Instead of looping through the errors to change the message we can simplify with:  `Joi.number().integer().min(1000000000).max(9999999999).required().messages({
  'number.min': 'The FRN is too short, it must be 10 digits',
  'number.max': 'The FRN is too long, it must be 10 digits',
  ...
})`

Reply:
Done